modin/pandas/base.py

```diff
@@ -2653,6 +2653,22 @@ def tz_localize(
         )
         return self.set_axis(labels=new_labels, axis=axis, inplace=not copy)
 
+    def value_counts(
```

Comment:
This is probably unnecessary if you rewrite `value_counts` in `dataframe.py` and `series.py`.